.github/workflows/mypy.yml

```diff
@@ -4,16 +4,8 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
-      - name: Download submodules
-        run: git submodule update --init --recursive
-      - name: Set up Python
-        uses: actions/setup-python@v1
+      - uses: actions/checkout@v3
         with:
-          python-version: 3.8
-      - name: Install dependencies
-        run: |
-          python -m pip install --upgrade pip
-          pip install mypy
+          submodules: recursive
       - name: Lint with mypy
-        run: mypy -p lark || true
+        run: pipx run tox -e type
```

Comment:
Magic!

Reply:
I _love_ that GHA supports pipx as a first-class package manager!